compiler/bindings/python/test/tools/compiler_tflite_test.py

```diff
@@ -48,10 +59,12 @@ def testImportBinaryPbFileOutputFile(self):
                     path, import_only=True, output_file=f.name
                 )
                 self.assertIsNone(output)
-                with open(f.name, "rt") as f_read:
-                    text = f_read.read()
+                with open(f.name, "rb") as f_read:
+                    bytecode = f_read.read()
             finally:
                 os.remove(f.name)
+        text = mlir_bytecode_to_text(bytecode)
+        print(text)
         logging.info("%s", text)
```

Comment:
No need to `print()` if already using `logging`, especially with `logging.basicConfig(level=logging.DEBUG)` at the bottom of the file

Reply:
Oh this is a leftover debug statement, thanks for spotting this! I will remove it